doc/contributing/security-release-process.md

```diff
@@ -159,8 +159,9 @@ security announcements for more information.
     folder and merging the PR.
 
 * [ ] 2\. **Cleanup:**
-  * [ ] Close PRs and backports.
-  * [ ] Close HackerOne reports:
+  * [ ] `git node security --cleanup`. This command will:
+  * Close PRs and backports.
+  * Close HackerOne reports:
```

Comment:
Optional nit that I think simplifies layout without sacrificing clarity. (But it's a nit, so disregard if you disagree!)

```suggestion
  * [ ] `git node security --cleanup`. This command will close PRs and backports.
    It will also close HackerOne reports:
```